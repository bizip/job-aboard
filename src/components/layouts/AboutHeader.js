import React from 'react';
import classes from './Header.module.css';

const propTypes = {};

const defaultProps = {};

const AboutHeader = () => (
  <div className={classes.aboutheader}>
    <h1>Don&#8217;t wait to apply as long as you fit</h1>
  </div>
);

AboutHeader.propTypes = propTypes;
AboutHeader.defaultProps = defaultProps;
// #endregion

export default AboutHeader;
