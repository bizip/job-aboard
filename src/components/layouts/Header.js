import React from 'react';
import classes from './Header.module.css';

const propTypes = {};

const defaultProps = {};

const Header = () => (
  <div className={classes.header}>
    <h1>Connect your talent to the job market</h1>
  </div>
);

Header.propTypes = propTypes;
Header.defaultProps = defaultProps;
// #endregion

export default Header;
