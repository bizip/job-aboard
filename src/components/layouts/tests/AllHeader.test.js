import renderer from 'react-test-renderer';
import Header from '../Header';
import AboutHeader from '../AboutHeader';

describe('Test for all header components', () => {
  it('It should render Header component', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('It should render About Header component', () => {
    const tree = renderer.create(<AboutHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
