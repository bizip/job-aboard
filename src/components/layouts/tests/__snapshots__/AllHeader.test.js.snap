// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for all header components It should render About Header component 1`] = `
<div
  className="aboutheader"
>
  <h1>
    Donâ€™t wait to apply as long as you fit
  </h1>
</div>
`;

exports[`Test for all header components It should render Header component 1`] = `
<div
  className="header"
>
  <h1>
    Connect your talent to the job market
  </h1>
</div>
`;
